dsew_community_profile/delphi_dsew_community_profile/pull.py

```diff
@@ -359,7 +372,7 @@ def fetch_new_reports(params, logger=None):
 
     # download and parse individual reports
     datasets = download_and_parse(listing, logger)
-
+    print(datasets, datasets.items())
```

Comment:
Is this a debugging leftover? Usually we try and use the structured logger to avoid mucking up the kibana dashboards

Reply:
Ah, I'll make it a logger command (but maybe @nmdefries can weigh in ) - this is because only a pdf instead of an excel file was uploaded on Jan 25/26th to the CDC site, and thus returned None for dataset items). Is it worth keeping in and investigating?

Reply:
If we want to fail in that case with a useful message, we can add an assert. But it would be reasonable to check if a given listing is empty in the `for sig, lst in datasets.items():` loop below. If it is empty, `continue` to the next listing.

Reply:
I think because it's not happened again, I'm going to just ignore it for now! That makes sense if it ever comes up again.